test/HostActivationTests/project.json

```diff
@@ -1,16 +1,15 @@
 {
   "version": "1.0.0-*",
-
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-004338"
+      "version": "1.0.0"
     },
```

Comment:
The test projects need to run against the sharedfx in the sdk, so `1.0.0` is right here. The `TestAssets` will run against the sharedfx produced, and there is xunit scenario coverage.